x/permission/keeper/keeper.go

```diff
@@ -283,14 +296,21 @@ func (k Keeper) VerifyPolicy(ctx sdk.Context, resourceID math.Uint, resourceType
 			effect, newPolicy = p.Eval(action, ctx.BlockTime(), opts)
 			if effect != types.EFFECT_UNSPECIFIED {
 				// check the operator is the member of this group
-				_, memberFound := k.GetGroupMember(ctx, item.GroupId, operator)
+				groupMember, memberFound := k.GetGroupMember(ctx, item.GroupId, operator)
 				if memberFound {
 					if effect == types.EFFECT_ALLOW {
 						allowed = true
 					} else if effect == types.EFFECT_DENY {
 						return types.EFFECT_DENY
 					}
 				}
+
+				// post check if the operator has not been expired
```

Comment:
is it possble that `ExpirationTime` is not nil, but with empty vaule?

Reply:
after the refactoring, the `ExpirationTime ` would not be `nil` it's only be set from user.